python/python/glide/async_commands/sorted_set.py

```diff
@@ -169,6 +170,145 @@ def __init__(
+        - Valid latitudes are from -85.05112878 to 85.05112878 degrees.
+
+    Args:
+        longitude (float): The longitude coordinate.
+        latitude (float): The latitude coordinate.
+    """
+
+    def __init__(self, longitude: float, latitude: float):
+        self.longitude = longitude
+        self.latitude = latitude
+
+
+class GeoUnit(Enum):
+    """
+    Enumeration representing distance units options for the `GEODIST` command.
+    """
+
+    METERS = "m"
+    """
+    Represents distance in meters.
+    """
+    KILOMETERS = "km"
+    """
+    Represents distance in kilometers.
+    """
+    MILES = "mi"
+    """
+    Represents distance in miles.
+    """
+    FEET = "ft"
+    """
+    Represents distance in feet.
+    """
+
+
+class OrderBy(Enum):
+    """
+    Enumeration representing sorting order options for `GEOSEARCH` command.
+    """
+
+    ASCENDING = "ASC"
+    """
+    Sort returned items from the nearest to the farthest, relative to the center point.
+    """
+    DESCENDING = "DESC"
+    """
+    Sort returned items from the farthest to the nearest, relative to the center point.
+    """
+
+
+class GeoSearchByRadius:
+    """
+    Represents search criteria of searching within a certain radius from a specified point.
+
+    Args:
+        radius (float): Radius of the search area.
+        unit (GeoUnit): Unit of the radius. See `GeoUnit`.
+    """
+
+    def __init__(self, radius: float, unit: GeoUnit):
+        """
+        Initialize the search criteria.
+        """
+        self.radius = radius
+        self.unit = unit
+
+    def __str__(self) -> str:
+        """
+        Convert the search criteria to the corresponding part of the Redis command.
+
+        Returns:
+            str: String representation of the search criteria.
+        """
+        return f"BYRADIUS {self.radius} {self.unit.value}"
+
+
+class GeoSearchByBox:
+    """
+    Represents search criteria of searching within a specified rectangular area.
+
+    Args:
+        width (float): Width of the bounding box.
+        height (float): Height of the bounding box
+        unit (GeoUnit): Unit of the radius. See `GeoUnit`.
+    """
+
+    def __init__(self, width: float, height: float, unit: GeoUnit):
+        """
+        Initialize the search criteria.
+        """
+        self.width = width
+        self.height = height
+        self.unit = unit
+
+    def __str__(self) -> str:
+        """
+        Convert the search criteria to the corresponding part of the Redis command.
+
+        Returns:
+            str: String representation of the search criteria.
+        """
+        return f"BYBOX {self.width} {self.height} {self.unit.value}"
+
+
+class GeoSearchCount:
+    """
+    Represents the count option for limiting the number of results in a GeoSearch.
+
+    Args:
+        count (int): The maximum number of results to return.
+        any_option (bool): Whether to allow returning as enough matches are found.
```

Comment:
```suggestion
        match_any (bool): Whether to allow for count approximately close matches.
```

Reply:
any means it dosent find all of them and then returns count of the result
it just finds the first count it can